docker-compose.yaml

```diff
@@ -1,32 +1,32 @@
-version: '2'
+version: '3.2'
 
 services:
-  service_php:
+  php:
     build: docker/php
-    tty: true
-    labels:
-      kompose.service.type: nodeport
     ports:
       - 9000:9000
     volumes:
       - ./quarto:/quarto
-
-  service_nginx:
+    depends_on:
+      - postgres
+  nginx:
     image: nginx:alpine
     ports:
       - "8081:8081"
     volumes:
       - ./quarto:/quarto
       - ./docker/nginx/site.conf:/etc/nginx/conf.d/default.conf
-
-  service_postgres:
+    depends_on:
+      - php
+  postgres:
     image: postgres:9.6
     environment:
-      - POSTGRES_USER=docker
-      - POSTGRES_PASSWORD=docker
+      - POSTGRES_USER=quatro
+      - POSTGRES_PASSWORD=quatro
+      - POSTGRES_DB=quatro
     ports:
         - "5433:5432"
-    expose:
-      - 5433
     volumes:
-      - ./quarto/var/data:/var/lib/postgresql/data
\ No newline at end of file
+      - quatro-pg-data:/var/lib/postgresql/data
```

Comment:
Docker allows [volume](https://docs.docker.com/compose/compose-file/#volumes) to be used without going through a mount point with the host, but to ensure data persistence.